packages/coordinator/src/services/verification/certificate-revocation/utils.ts

```diff
@@ -56,9 +57,11 @@ const getPrescriptionId = (parentPrescription: hl7V3.ParentPrescription): string
   return parentPrescription.id._attributes.root
 }
 
-const getRevokedCertReasonCode = (cert: RevokedCertificate): number => {
-  const crlExtension = cert.crlEntryExtensions?.extensions.find(ext => ext.extnID === CRL_REASON_CODE_EXTENSION)
-  return crlExtension ? parseInt(crlExtension.parsedValue.valueBlock) : null
+const getRevokedCertReasonCode = (cert: X509CrlEntry | X509Certificate): number => {
```

Comment:
I would have thought cert could be just X509CrlEntry, I can't see why it would need to be X509Certificate